Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -268,6 +268,8 @@ public struct ApolloCodegenConfiguration {
   /// See `APQConfig` for more information on Automatic Persisted Queries.
   public let apqs: APQConfig
 
+  public let experimentalClientControlledNullability: Bool
```

Comment:
We might want to move this into its own struct to set us up for other experimental features that might crop up in the future.